front/app/modules/commercial/native_surveys/components/NativeSurveyForm/NativeSurveyForm.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
```

Comment:
Same suggestion on renaming this to `index.tsx`. Could be applied for other files as-well